src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+import main.java.*;
```

Comment:
Should the classes in main.java be imported explicitly instead of a wildcard import?

Reply:
Yes, he should use single class imports instead of wildcards